test_e2e/dest/duckdb.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
We've recently updated to add integration tests rather than e2e tests for new connectors, take a look at the s3 example: [test_s3.py](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/test/integration/connectors/test_s3.py). Could we drop this file and add a new integration test instead? That should help isolate just the code being added in this PR.